shell/artifacts/Social/Particles.manifest

```diff
@@ -23,16 +23,27 @@ particle OnlyShowPosts in 'source/ShowPosts.js'
   description `show ${posts}`
 
 particle ShowSinglePost in 'source/ShowSinglePost.js'
-  ShowSinglePost(in Post post)
-  consume root
+  ShowSinglePost(in Post post, in [Avatar] avatars)
+  consume item
   description `show ${post}`
 
 particle WritePosts in 'source/WritePosts.js'
   WritePosts(inout Post post, inout [Post] posts)
   consume root
   description `write posts`
 
+// TODO(wkorman): Should we just use HostedParticleShape as
+// defined in Multiplexer.manifest directly?
+shape RenderParticleShape
+  RenderParticleShape(in Post)
+  consume
+
 particle EditPost in 'source/EditPost.js'
-  EditPost(inout Post post, inout [Post] posts, in Person user)
+  EditPost(host RenderParticleShape renderParticle, inout Post post, inout [Post] posts, in Person user)
   consume content
   description `edit a post`
+
+particle PostMuxer in 'source/PostMuxer.js'
+  PostMuxer(in [Post] list, in [Avatar] avatars)
```

Comment:
I'm going to explore whether we can do without the `avatars` field here in the muxer, and mapping `#BOXED_avatars` in the `Social.recipes` associated recipe as well, in a separate PR.